src/shared/components/InputBase/InputBase.styles.ts

```diff
@@ -1,101 +1,67 @@
 import { colors, sizes, transitions, typography } from '@/shared/theme'
-import { css } from '@emotion/react'
 import styled from '@emotion/styled'
 
 type FormGroupProps = {
   error?: boolean
   disabled?: boolean
 }
 
-export const FormGroup = styled.div<FormGroupProps>`
-  position: relative;
+export const FormGroup = styled.label<FormGroupProps>`
```

Comment:
What's the rationale for using `label` as a container vs just for the label text itself?

Reply:
If you use the `label` as a wrapper for `input`, you don't have to add `htmlFor/for` attribute to `label` and `id` to `input`.

Reply:
Oh, cool, I wasn't aware :+1: